getWordCount.ts

```diff
@@ -1,23 +1,28 @@
-    .source;
-const nonSpaceDelimitedWords = /\u3041-\u3096\u309D-\u309F\u30A1-\u30FA\u30FC-\u30FF\u4E00-\u9FD5/
-    .source;
+export default function getWordCount(text: string): number {
+  const spaceDelimitedChars =
```

Comment:
it's nothing of your responsibility, but making sense of these diff is hard for people's eyes;

what was exactly added to cover the cases of delimiters? only apostrophe?

Reply:
I didn't add anything, I treated it like machine code :D

I simply copied the latest changes from the `utils.ts` file.